### ISSUE TITLE
View, Search and Filter Change Log

### ISSUE THREAD

**Issue by ashley-dhillon** [bcgov/parks-data-register#31]
As a...
I need...
So that...

Acceptance Criteria:

Notes:

**Comment by JLWade** [comment]
unblocking because #230 has been reviewed

**Comment by cameronpettit** [comment]
From the PR: 

>This change adds a changelog search to the front end. As per the ticket, the first 20 results are loaded in descending chronological order when landing on the page. Filtering by text or by statusChanged or legalNameChanged is also possible.

>Note that only changelog items that have been created after #318 was merged will have the correct structure to appear properly in the changelog search. If there are existing changelog items that should be shown, we should create a new ticket to migrate them to the proper format.

**Comment by manuji** [comment]
Tested on TEST: In progress. 

* Tested and verified that the feature has been implemented. 
* Separate tickets have been created for the bugs.

Linked bugs:
* https://github.com/bcgov/parks-data-register/issues/345

**Comment by manuji** [comment]
@janemoxd , I have done the functional testing and now waiting for the bug fixes. This ticket can be UX-reviewed whenever you have time.

**Comment by JLWade** [comment]
Moving to done - and UI tickets in next sprint
